go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.9.0
-	github.com/containers/buildah v1.19.7
```

Comment:
This will give us a higher version of Buildah vendored into Podman than we have available on RHEL (v1.19.6).

Reply:
Not sure what that implies.  We need `.7` to fix the issue.

Reply:
Just concerned that if you run Buildah you could get different behavior than running `podman build`.  I'm probably over-fretting.